frontend/src/Components/forms/LoginForm.tsx

```diff
@@ -74,6 +74,7 @@ export default function LoginForm() {
                     return;
                 }
             } catch (error: any) {
+                //Todo: unsure about what to type this as Error or ApiError
```

Comment:
So let me make sure that I understand this correctly. when there is an error, and there  is a 401 or 403 the response sent back to the LoginForm.tsx will be that returned from line 20 and in all other cases of an error, it will return the object that is being passed into the reject on line 22?
So when I create the interface for that shape in the common.ts and use that type to replace the any that is on the LoginForm.tsx I get the error below.

![image](https://github.com/user-attachments/assets/4cc174db-1511-437b-8613-031e469254eb)


![image](https://github.com/user-attachments/assets/e68e15f2-e5be-453f-861e-8000499fbc8b)